lib/erlef/accounts/member.ex

```diff
@@ -59,7 +62,12 @@ defmodule Erlef.Accounts.Member do
     field(:has_requested_slack_invite, :boolean, default: false)
     field(:requested_slack_invite_at, :utc_datetime)
     field(:deactivated_at, :utc_datetime)
+
     embeds_one(:external, Erlef.Accounts.External, on_replace: :update)
+
+    belongs_to(:sponsor, Sponsor)
```

Comment:
@starbelly I suspect the common access pattern would be `member -> sponsor` because we fetch the member from the database on every request and the member is the one performing the create/update/delete actions on the job posts.